src/Symfony/Component/Mime/Part/TextPart.php

```diff
@@ -146,6 +146,14 @@ public function getPreparedHeaders(): Headers
         return $headers;
     }
 
+    /**
+     * @internal
```

Comment:
it doesn't make sense to me to have an internal method for cross component uses
if we think this is a needed API, we should commit to it

Reply:
I'm against making this public, that leaks internal information I don't want to expose.